apis/gitops.turbonomic.io/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha1 contains API Schema definitions for the gitops.turbonomic.io v1alpha1 API group
+// +kubebuilder:object:generate=true
+// +groupName=gitops.turbonomic.io.turbonomic.io
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "gitops.turbonomic.io.turbonomic.io", Version: "v1alpha1"}
```

Comment:
"gitops.turbonomic.io.turbonomic.io". I don't think we want this as group.

Reply:
I hadn't noticed that. I'll look into why that was duplicated.

Reply:
Looks like that was the result of passing `--group gitops.turbonomic.io` to the `kubebuilder create...` command. Should have just passed `--group gitops`.

Reply:
As far as moving to a `devops` group, I think that makes sense. I'll update it.

Reply:
Updated to be within the `devops` group.